src/converters/dictionaries.cc

```diff
@@ -760,3 +807,21 @@ TO_JS_IMPL(webrtc::VideoFrame, value) {
   frame->Set(Nan::New("data").ToLocalChecked(), data);
   return node_webrtc::Pure(scope.Escape(frame).As<v8::Value>());
 }
+
+TO_JS_IMPL(node_webrtc::RTCOnDataEventDict, dict) {
+  Nan::EscapableHandleScope scope;
+
+  auto isolate = Nan::GetCurrentContext()->GetIsolate();
+  auto byteLength = dict.numberOfChannels * dict.numberOfFrames * dict.bitsPerSample / 8;
+  auto arrayBuffer = v8::ArrayBuffer::New(isolate, dict.audioData.get(), byteLength, v8::ArrayBufferCreationMode::kInternalized);
+  auto uint8Array = v8::Uint8ClampedArray::New(arrayBuffer, 0, byteLength);
+  dict.audioData.reset();
```

Comment:
So, converting an `node_webrtc::RTCOnDataEventDict` to a `v8::Local<v8::Value>` will take ownership of the underlying `std::shared_ptr<uint8_t>` and set it to `nullptr` on the `node_webrtc::RTCOnDataEventDict`.

Reply:
Turns out `reset` doesn't do what I want.